app/frontend/src/components/form/RequestReceipt.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <v-btn
+      color="primary"
+      class="mx-5 mb-10"
+      :disabled="success"
+      fab
+      large
+      v-on:click="displayDialog()"
+    >
+      <v-icon v-if="success">check</v-icon>
+      <v-icon v-else>email</v-icon>
+    </v-btn>
+
+    <BaseDialog
+      :show="showDialog"
+      type="CONTINUE"
+      @close-dialog="showDialog = false"
+      @continue-dialog="requestReceipt()"
+    >
+      <template v-slot:icon>
+        <v-icon large color="primary">email</v-icon>
+      </template>
+      <template v-slot:text>
+        <label>Send to E-mail Address</label>
```

Comment:
Could add a form tag around and set lazy validation. I noticed if I clear the email field as soon as I start typing it yells at me for entering an invalid email even though I'm not done entering what will be a valid email.
Also, if I do enter "sdfsdfd" for the email address it lets me submit the form, that can be blocked by validating the form ref on requestReceipt().

Reply:
I've added a form around (without lazy-validation as discussed) to prevent users from submitting garbage emails (and then getting a 422 failure).